partner_duplicate_mgmt/README.rst

```diff
@@ -53,6 +53,22 @@ A new list view shows alls duplicates. The user can:
 * exclude the duplicate by choice 
 
 
+Company and contact merger
+--------------------------
+Not possible
+
+
+Companies merger
+----------------
+Possible
+
+
+Contacts merger
+---------------
+To merge contacts that are linked to journal items, the user must be part of the group
+"Contacts Merge With Account Moves", if not the merger won't be possible.
```

Comment:
The name of the section should be something like:

Merging Partners with Account Moves
----------------------------------------------
...


The reason is that this feature is unrelated to the partner being a contact or a company. Also, the name of the group is incorrect. It should be "Partner Merge With Account Moves". If the 2 partners have account moves, the user must be part of the group in order to merge these partners.